scripts/automator/build/os-darwin

```diff
@@ -2,9 +2,11 @@
 ar="ar"
 ranlib="ranlib"
 function make_binary() {
+    # Limit GCC to ASCII-only characters
+    LC_ALL=C LANG=en_US LANGUAGE=en_US \
```

Comment:
I believe it can fail on really, really minimal Debian system, but I wouldn't worry about it :)

I installed Debian only with C and en_GB once. It was fun (not).